frontend/src/pushapi.js

```diff
@@ -21,49 +17,16 @@ angular.module('clouway-push', [])
     };
 
     /**
-     * Set a method to call for opening of connection.
-     * The method must return a promise.
+     * Sets the backend url connection.
      *
-     * @param {function(subscriber)} method method that will be called for opening of connection.
+     * @param url the new url
      * @returns {*} the provider instance for chaining purposes.
      */
-    this.openConnectionMethod = function (method) {
-      this.connectionMethods.connect = method;
+    this.backendServiceUrl = function(url) {
```

Comment:
I've excluded pushapi-mocks here because it will prevent testing of the pushApi itself.

pushapi-mocks is supposed to be used in an app(let's call it `deltaApp`) that uses pushApi to simplify deltaApp's tests.